atc/lidar/checker.go

```diff
@@ -25,6 +27,8 @@ type checker struct {
 
 	checkFactory db.CheckFactory
 	engine       engine.Engine
+
+	running *sync.Map
```

Comment:
what is the memory impact for using this map? Is it possible that same check got picked up by different ATCs that creates overheads in memory?

Reply:
This change is actually to limit the amount of memory used by each ATC by making sure we don't run duplicate checks.